src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -22,16 +22,19 @@ import com.google.cloud.pubsub.v1.Publisher
 import com.google.protobuf.ByteString
 import com.google.pubsub.v1.PubsubMessage
 import com.google.pubsub.v1.TopicName
+import org.slf4j.LoggerFactory
 
-fun getResultData(taskId: String, statusCode: Int, resultBytes: ByteArray): ByteString {
+fun getResultData(taskId: String, statusCode: Int, result: String): ByteString {
     return CompilerFleet.CompilerFleetResult.newBuilder()
             .setTaskId(taskId)
             .setStatusCode(statusCode)
-            .setResultBytes(ByteString.copyFrom(resultBytes))
+            .setResultBytes(ByteString.copyFrom(result.toByteArray()))
             .build()
             .toByteString()
 }
 
+private val logger = LoggerFactory.getLogger("Publisher")
```

Comment:
We usually write logger in uppercase: `LOGGER`.

Reply:
Done